moped-editor/src/views/projects/projectView/ProjectComponents/ComponentListItem.js

```diff
@@ -43,6 +43,15 @@ export default function ComponentListItem({
   const onStartEditingComponent = () =>
     editDispatch({ type: "start_edit", payload: component });
 
+  const isSignalComponent =
+    component?.moped_components?.feature_layer?.internal_table ===
+    "feature_signals";
+  const componentName = component?.moped_components?.component_name;
+  const signalLocationName = component?.feature_signals?.[0]?.location_name;
+  const listItemPrimaryText = isSignalComponent
+    ? `${componentName} -${signalLocationName}`
+    : componentName;
```

Comment:
Thought about wrapping `ComponentListItem` with another component like `SignalComponentListItem` and moving this logic there, but just decided to ship this. If anyone feels strongly about refactoring, I'll update for sure. 🙏

Reply:
AMD will be very happy added the location name here - nice! i think this is fine for now. we're going to have to revisit the list item styles as we add additional component properties, e.g. [component phasing](https://github.com/cityofaustin/atd-data-tech/issues/10680).